homeworks/src/vkochergin/module1/example/Homework3/HackerRankDay4.java

```diff
@@ -0,0 +1,37 @@
+package vkochergin.module1.example.Homework3;
+
+/**
+ * Created by Volodymyr Kochergin on 14.04.2017.
+ */
+public class HackerRankDay4 {

```

Comment:
https://github.com/PBaranovskyi/learningJava/pull/24/commits/6127fc2ef20d4c3672ae5c07dd86e79d69e9db4e commit with main method is added